core/metamodel/src/main/java/org/apache/isis/metamodel/adapter/oid/Oid_Marshaller.java

```diff
@@ -120,12 +118,6 @@ private Oid_Marshaller(){}
                             ")" +
                             ")" +
                             "(" + "[" + SEPARATOR_PARENTED + "]" + WORD + ")?"  + // optional collection name
-                            "(" +
```

Comment:
It might be worthwhile keeping "(.*)?" at the end, as an unnamed group. That would allow any legacy OIDs that might be bookmarked or persisted to still be interpreted as a valid Oid ... the version info will just be thrown away

Reply:
done